experimental/packages/sampler-jaeger-remote/package.json

```diff
@@ -48,6 +48,7 @@
     "@opentelemetry/api": "^1.3.0"
   },
   "devDependencies": {
+    "@opentelemetry/api": "1.9.0",
```

Comment:
I guess I would have figured that the peerDep would suffice here.  From `npm help json` section on `peerDependencies`:

```
       In npm versions 3 through 6, peerDependencies were not automatically
       installed, and would raise a warning if an invalid version of the peer
       dependency was found in the tree. As of npm v7, peerDependencies are
       installed by default.
```

npm@7 was the default npm version as of Node.js v15.